Extensions/AdvancedWindow/JsExtension.js

```diff
@@ -578,16 +589,14 @@ module.exports = {
       .addAction(
         'SetOpacity',
         _('Set window opacity'),
-        _(
-          'Changes the window opacity. The new opacity should be between 0 and 1.'
-        ),
+        _('Changes the window opacity.'),
         _('Set the window opacity to _PARAM0_'),
         _('Advanced window management/Windows, Linux, macOS'),
         'res/actions/window24.png',
         'res/actions/window.png'
       )
-      .addParameter('number', _('New opacity'), '', false)
-      .setDefaultValue('true')
+      .addParameter('expression', _('New opacity'), '', false)
+      .setParameterLongDescription('This has to be a float from 0 to 1.')
```

Comment:
`float` is a technical term that we usually don't surface in the GDevelop interface.

`A number between 0 (fully transparent) and 1 (fully opaque).`